modules/020-cni-cilium/templates/agent/rbac-to-us.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: access-to-agent-agent-prometheus-metrics
+  namespace: d8-cni-cilium
+  {{- include "helm_lib_module_labels" (list . (dict "app" "node-exporter")) | nindent 2 }}
+rules:
+- apiGroups: ["apps"]
+  resources: ["daemonsets/prometheus-metrics"]
+  resourceNames: ["agent"]
+  verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: access-to-agent-agent-prometheus-metrics
+  namespace: d8-cni-cilium
+  {{- include "helm_lib_module_labels" (list . (dict "app" "node-exporter")) | nindent 2 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: access-to-agent-agent-prometheus-metrics
+subjects:
+- kind: User
```

Comment:
There are more subjects to add right now. Please search for an example in other modules.

Reply:
Done.